frontend/src/components/object/DeleteObjectButton.vue

```diff
@@ -37,13 +37,15 @@ const confirmDelete = () => {
     const msgContext = props.ids.length > 1 ? `the selected ${props.ids.length} ${item}s` : `this ${item}`;
     confirm.require({
       message: `Please confirm that you want to delete ${msgContext}.`,
-      header: `Delete ${props.ids.length > 1 ? item + 's' : item }`,
+      header: `Delete ${props.ids.length > 1 ? item + 's' : item}`,
       acceptLabel: 'Confirm',
       rejectLabel: 'Cancel',
       accept: async () => {
         try {
-          await objectStore.deleteObjects(props.ids, props.versionId);
-          emit('on-deleted-success', props.versionId);
+          props.ids?.forEach(async (ele) => {
+            const res = await objectStore.deleteObjects([ele], props.versionId);
+            if (res) emit('on-deleted-success', props.versionId);
+          });
```

Comment:
The way this is written, I suspect if there are multiple failures, this will not handle that as the next real catch halts the forEach iteration, so this is not truly asynchronous behavior.